src/components/CommentList/CommentList.jsx

```diff
@@ -1 +1,16 @@
-export const CommentList = () => <>Put the list here</>;
+import { CommentInfo } from '../CommentInfo/CommentInfo';
```

Comment:
1. you can write it shorter:

`import { CommentInfo } from '../CommentInfo/CommentInfo';` -> `import { CommentInfo } from '../CommentInfo';`

2. add empty line to group imports

Reply:
To short your imports you should create index.js file in your directory

Read more about index files:
https://dev.to/fahadaminshovon/-how-to-use-indexjs-fileproperly-302f